every_election/apps/organisations/boundaries/boundaryline.py

```
@@ -57,6 +57,23 @@ def get_feature_by_field(self, fieldname, code):
         """
         return self.merge_features(matches)
 
+    def get_feature_by_name_and_county(self, tuple):
+        matches = []
+        for feat in self.layer:
+            feat_name = normalize_name_for_matching(feat.get("name"))
+            feat_county = normalize_name_for_matching(
+                feat.get("file_name")
+            ).replace("_", "-")
+            if (feat_name, feat_county) == tuple:
+                matches.append(feat)
+
+        if len(matches) == 0:
+            raise ObjectDoesNotExist(f"Expected one match for {tuple}, found 0")
+        if len(matches) == 1:
+            return matches[0].geom.geos
+
+        return self.merge_features(matches)
+
     def get_code_from_feature(self, feature):
         if feature.get("area_code") == "CED":
             return "unit_id:" + str(feature.get("unit_id"))
```

Lets not do anything about it in this PR, but while I'm seeing this code.. lets :round_pushpin: put a pin it it to come back to when we look at https://app.asana.com/0/1209090479296269/1209157822204377/f

---

I've started working on that ticket here: #2400. But wasn't sure what to do about these lines.

---

I think we can just remove them and let it throw (so we follow the `if feature.get("code") == "999999999":` branch)

The only place we call this is from `boundaryline_backport_codes` (which isn't the right command to use for CEDs). So if we're trying to use that command to attach codes to CEDs then its good if we throw here.